pkg/client/status.go

```diff
@@ -0,0 +1,57 @@
+package client
+
+import (
+	"context"
+)
+
+type Status struct {
```

Comment:
`Status` is an extremely generic name. Paperless' code uses `SystemStatus` (https://github.com/paperless-ngx/paperless-ngx/blob/3104417076245027f85e18e67f4278609e3e49ec/src/paperless/urls.py#L194-L196) which is more specific.

Proposed names for the structures:

* `SystemStatus`
* `SystemStatusStorage`
* `SystemStatusDatabase`
* `SystemStatusDatabaseMigration`
* `SystemStatusTasks`

Reply:
Done in https://github.com/hansmi/paperhooks/pull/44/commits/50d0d8dc34669ea2b2edbb54b7e0b5faa3074b29